farm-bot/codebase/contracts/farm-bot.sol

```diff
@@ -42,21 +42,23 @@ contract FarmBot is Owned, FarmbotERC20 {
     constructor(
         address _owner,
         address _stakingRewards,
-	address _router,
-	address[] memory _path0,
-	address[] memory _path1
+		address _router,
+		address[] memory _path0,
+		address[] memory _path1,
+		string memory _symbol
     ) Owned(_owner) {
         stakingRewards = StakingRewards(_stakingRewards);
-	rewardsToken = stakingRewards.rewardsToken();
+		rewardsToken = stakingRewards.rewardsToken();
```

Comment:
nit: can we remove those whitespace here?

Reply:
so weird. Remix and Webstorm both show this as looking totally normal, but github and textedit make it look terrible